### PR TITLE
Created an Issue template

### PR DESCRIPTION
**Pull request by adityaraute**

### CODE REVIEW

File: .github/workflows/ISSUE_TEMPLATES/feature.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+name: Feature Request
+about: To request a feature that would enhance the bot's working
```
Comment (BOLT04) [inline .github/workflows/ISSUE_TEMPLATES/feature.md]: what do you think of changing the text from bot, to something more generic?
```suggestion
about: To request a feature that would enhance the project
```

**Review by BOLT04 — COMMENTED** [review]
Thank you for moving the PR to this repo. Just one comment, but looks good 👍

**Review by BOLT04 — APPROVED** [review]
LGTM 👍, thank you @dumbcoder7 for adding this template.

**Comment by eddiejaoude** [comment]
Thank you both 👍

**Pull request merged**